### ISSUE TITLE
core_load causes crash on launch

### ISSUE THREAD

**Issue by ojsl1** [flightlessmango/MangoHud#667]
After enabling load by core option from goverlay:
```
(Fri–22:32) otter ~ $ goverlay
QWidget::paintEngine: Should no longer be called
QWidget::paintEngine: Should no longer be called
[2022-01-07 22:32:54.214] [MANGOHUD] [info] [config.cpp:112] skipping config: '/usr/lib/MangoHud.conf' [ not found ]
[2022-01-07 22:32:54.214] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/goverlay.conf' [ not found ]
[2022-01-07 22:32:54.214] [MANGOHUD] [info] [config.cpp:117] parsing config: '/home/otter/.config/MangoHud/MangoHud.conf'
[2022-01-07 22:32:54.355] [MANGOHUD] [error] [cpu.cpp:553] Failed to initialize CPU power data
[2022-01-07 22:32:54.367] [MANGOHUD] [info] [overlay.cpp:697] Uploading is disabled (permit_upload = 0)
[2022-01-07 22:32:54.390] [MANGOHUD] [info] [imgui_impl_opengl3.cpp:418] GL version: 4.6
paint
terminate called after throwing an instance of 'std::invalid_argument'
  what():  stoi
/usr/bin/goverlay: line 9:  3616 Aborted                 (core dumped) mangohud --dlsym /usr/lib/goverlay --style Breeze
```

Lutris logs after launching Overwatch from Battle.net client:
```
[2022-01-07 22:36:20.292] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine-Overwatch.conf' [ not found ]
[2022-01-07 22:36:20.292] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.local/share/lutris/runners/wine/lutris-6.21-6-x86_64/bin/MangoHud.conf' [ not found ]
[2022-01-07 22:36:20.292] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine64-preloader.conf' [ not found ]
[2022-01-07 22:36:20.292] [MANGOHUD] [info] [config.cpp:117] parsing config: '/home/otter/.config/MangoHud/MangoHud.conf'
[2022-01-07 22:36:20.293] [MANGOHUD] [error] [cpu.cpp:553] Failed to initialize CPU power data
[2022-01-07 22:36:20.559] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine-Overwatch.conf' [ not found ]
[2022-01-07 22:36:20.559] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.local/share/lutris/runners/wine/lutris-6.21-6-x86_64/bin/MangoHud.conf' [ not found ]
[2022-01-07 22:36:20.559] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine64-preloader.conf' [ not found ]
[2022-01-07 22:36:20.559] [MANGOHUD] [info] [config.cpp:117] parsing config: '/home/otter/.config/MangoHud/MangoHud.conf'
[2022-01-07 22:36:20.564] [MANGOHUD] [error] [cpu.cpp:553] Failed to initialize CPU power data
[2022-01-07 22:36:22.488] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine-Overwatch.conf' [ not found ]
[2022-01-07 22:36:22.488] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.local/share/lutris/runners/wine/lutris-6.21-6-x86_64/bin/MangoHud.conf' [ not found ]
[2022-01-07 22:36:22.488] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine64-preloader.conf' [ not found ]
[2022-01-07 22:36:22.488] [MANGOHUD] [info] [config.cpp:117] parsing config: '/home/otter/.config/MangoHud/MangoHud.conf'
[2022-01-07 22:36:22.488] [MANGOHUD] [error] [cpu.cpp:553] Failed to initialize CPU power data
[2022-01-07 22:36:22.522] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine-Overwatch.conf' [ not found ]
[2022-01-07 22:36:22.522] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.local/share/lutris/runners/wine/lutris-6.21-6-x86_64/bin/MangoHud.conf' [ not found ]
[2022-01-07 22:36:22.522] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine64-preloader.conf' [ not found ]
[2022-01-07 22:36:22.522] [MANGOHUD] [info] [config.cpp:117] parsing config: '/home/otter/.config/MangoHud/MangoHud.conf'
[2022-01-07 22:36:22.523] [MANGOHUD] [error] [cpu.cpp:553] Failed to initialize CPU power data
[2022-01-07 22:36:22.563] [MANGOHUD] [info] [overlay.cpp:697] Uploading is disabled (permit_upload = 0)
[2022-01-07 22:36:23.562] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine-Overwatch.conf' [ not found ]
[2022-01-07 22:36:23.562] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.local/share/lutris/runners/wine/lutris-6.21-6-x86_64/bin/MangoHud.conf' [ not found ]
[2022-01-07 22:36:23.562] [MANGOHUD] [info] [config.cpp:112] skipping config: '/home/otter/.config/MangoHud/wine64-preloader.conf' [ not found ]
[2022-01-07 22:36:23.562] [MANGOHUD] [info] [config.cpp:117] parsing config: '/home/otter/.config/MangoHud/MangoHud.conf'
[2022-01-07 22:36:23.562] [MANGOHUD] [error] [cpu.cpp:553] Failed to initialize CPU power data
[2022-01-07 22:36:23.596] [MANGOHUD] [info] [overlay.cpp:697] Uploading is disabled (permit_upload = 0)
terminate called after throwing an instance of 'std::invalid_argument'
  what():  stoi
*** stack smashing detected ***: terminated
```

~/.config/MangoHud/MangoHud.conf
```
################### File Generated by GOverlay ###################

toggle_fps_limit=F1


legacy_layout=false
gpu_stats
gpu_temp
gpu_text=GPU
cpu_stats
cpu_temp
core_load
cpu_color=2e97cb
cpu_text=CPU
io_color=a491d3
vram_color=ad64c1
ram
ram_color=c26693
fps
engine_version
engine_color=eb5b5b
gpu_color=2e9762
wine
wine_color=eb5b5b
frame_timing=1
frametime_color=00ff00
media_player_color=ffffff
time
background_alpha=0.4
font_size=24

background_color=020202
position=top-left
text_color=ffffff
round_corners=0
toggle_hud=Shift_R+F12
toggle_logging=Shift_L+F2
upload_log=F5
output_folder=/home/otter
media_player_name=spotify
```

Hardware information copied from lutris:
```
[System]
OS:              Arch Linux rolling n/a
Arch:            x86_64
Kernel:          5.15.13-arch1-1
Desktop:         Not found (Herbstluftwm)
Display Server:  x11

[CPU]
Vendor:          GenuineIntel
Model:           Intel(R) Core(TM)2 Quad CPU    Q9550  @ 2.83GHz
Physical cores:  4
Logical cores:   4

[Memory]
RAM:             7.8 GB
Swap:            0.0 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce GTX 770/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 470.94
OpenGL Core:     4.6.0 NVIDIA 470.94
OpenGL ES:       OpenGL ES 3.2 NVIDIA 470.94
Vulkan:          Supported
```

Should I upload the coredump? I don't know how to read gdb

**Comment by flightlessmango** [comment]
I think this has been fixed in master

**Comment by ojsl1** [comment]
Yep, fixed now as of 0.6.6.1.r23.g161081b-1 aur build, thanks!
